src/applications/static-pages/events/helpers/index.unit.spec.js

```diff
@@ -315,12 +315,6 @@ describe('deriveEventLocations', () => {
     expect(deriveEventLocations()).to.deep.equal([]);
   });
 
-  it('handles when fieldLocationHumanreadable is truthy', () => {
```

Comment:
Curious - why was this test removed?

Reply:
This helper no longer renders this field, its an optional field in the CMS